lib/binding.pry.rb

```diff
@@ -0,0 +1,15 @@
+require 'pry'
+
+binding_num = 2
+
+loop do
+  b    = binding.of_caller binding_num
+  iseq = b.instance_variable_get("@iseq")
+
+  if iseq.path[/kernel_require\.rb$/] and iseq.label == "require" # find the require method
```

Comment:
Good point... this probably needs some testing. :)

Old rubies might also have their `require` defined in a different file (rubygems.rb?)